Formula/massdns.rb

```diff
@@ -0,0 +1,30 @@
+class Massdns < Formula
+  desc "Particle library for 3D graphics"
```

Comment:
The description seems to be a copy paste error.

Reply:
Already fixed. ebc5b2f357be466fda209754b110e0daca8a49e4